task8/script.js

```diff
@@ -0,0 +1,61 @@
+const documentFragment = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
+
+const listItem = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+// ローディング画像を表示
+const insertLoading = () => {
+  const loading = document.getElementById("div");
+  const img = new Image();
+  img.src = "loading-circle.gif";
+  img.id = "loadingImg";
+  loading.appendChild(img);
+}
+
+// ローディング画像を非表示
```

Comment:
Same here.

Reply:
Same as the answer above

commit: 48729d2